core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -160,6 +160,95 @@ class TopicCommandTest extends ZooKeeperTestHarness with Logging with RackAwareT
     TopicCommand.createTopic(zkClient, createNotExistsOpts)
   }
 
+  @Test
+  def testCreateWithAllAvailableBrokersReplicaAssignment() {
```

Comment:
Can we also add any missing tests/scenarios to [AdminClientIntegrationTest ](https://github.com/apache/kafka/blob/trunk/core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala#L149)